helm/prometheus-rules/templates/kaas/phoenix/alerting-rules/irsa.rules.yml

```diff
@@ -0,0 +1,49 @@
+# This rule applies to vintage aws or capa management clusters
+{{- if or (eq .Values.managementCluster.provider.kind "aws") (eq .Values.managementCluster.provider.kind "capa") }}
```

Comment:
I think this is not respecting the multi-provider mc things if we run aws WCs from another provider 🤔

Reply:
@giantswarm/team-phoenix tis only runs on an AWS/CAPA MC or can it run on any other MCs if we run CAPA/EKS WCs?